### DISCUSSION TITLE
[Feature] Use trickplay files for better ML video analysis

### DISCUSSION THREAD

**Discussion by DerSpatz** [immich-app/immich#11092] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Jellyfin had a plugin called JellyScrub that is now merged into the official Jellyfin releases.

https://github.com/nicknsy/jellyscrub

JellyScrub generates a preview image of the video files every few seconds so that the user has a preview while scrubbing through the video playback.

This function could be used to generate more screenshots from video files that could be used internally for better ML analysis of the video files.

From my understanding, Immich currently uses only one thumbnail for analyzing video files. With additional screenshots, Immich would be able to also add contexts or persons that are only visible later in the video, which would make it much more useful for longer videos.

Bonus points if the search function then also jumps directly to the position of the video where the person ist first seen.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5936
